### ISSUE TITLE
Missing Variable for Directories in URLs in SCSS

### ISSUE THREAD

**Issue by rstropek** [telerik/kendo-theme-default#44]
### Repro steps
1. Create new Angular app with Angular CLI: `ng init --style scss`
1. Install Kendo them using NPM: `npm install --save @telerik/kendo-theme-default`
1. Import Kendo SCSS in `styles.scss`
    ```
    @import "../node_modules/@telerik/kendo-theme-default/styles/packages/all.scss";
    ```
1. Build the web app with `ng build` or run it with `ng serve`. **Webpack throws an error because it cannot find resources like `KendoUIGlyphs.woff` using relative path.**
1. Override `icon-font-dir` in `styles.scss`:
    ```
    $icon-font-dir: '../node_modules/@telerik/kendo-theme-default/styles/icons/font/' !default;
    @import "../node_modules/@telerik/kendo-theme-default/styles/packages/all.scss";
    ```
1. Build the web app with `ng build` or run it with `ng serve`. Webpack can now find the font. **However, `progressbar/_theme.scss` and `slider/_theme.scss` contain relativ paths without a variable that you could override.**

If one doesn't need progressbar or slider, she can switch from `all.scss` to more specific files (e.g. `grid.scss`, `layout.scss`). However, if you need the affected controls, there is no way to include the SCSS files in `styles.scss`.

### Proposed Solution
1. Add variable to all relative paths so that they can be overridden in `styles.scss`.
1. Extend [documentation](https://github.com/telerik/kendo-theme-default/blob/master/docs/index.md) mentioning how to include Kendo theme in `styles.scss`.

**Comment by gyoshev** [comment]
Thank you for the feedback, @rstropek. We have decided to inline the resources, which makes it much easier to use the theme (both with Angular CLI and in general). I'll post an update when this has been released.

**Comment by rstropek** [comment]
Thank your for your response. Do you have an approximate time frame? Rough estimation would be enough so that we know whether we have to implement some kind of workaround.

**Comment by gyoshev** [comment]
It should be in place by the end of the week.

**Comment by rstropek** [comment]
Awesome. Thank you!

**Comment by gyoshev** [comment]
The change has landed in [version 1.29.0 of the theme package](https://github.com/telerik/kendo-theme-default/releases/tag/v1.29.0). With it, a single-line import will include the complete theme:

`@import "~@telerik/kendo-theme-default/styles/packages/all";`

**Comment by rstropek** [comment]
Works perfectly fine, thank you!
